l10n_br_fiscal/views/payment_view.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 KMEE
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="l10n_br_fiscal_payment_form_view">
+        <field name="name">l10n_br_fiscal.payment.form (in l10n_br_fiscal)</field>
+        <field name="model">l10n_br_fiscal.payment</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <field name="document_id" invisible="1"/>
+                    <field name="company_id" invisible="1"/>
+                    <field name="currency_id" invisible="1"/>
+                    <group>
+                        <group>
+                            <field name="amount" required="1"/>
```

Comment:
Este campo traz sempre o valor cheio da nota fiscal sem levar em consideração o valor dos demais tipos de pagamento já definidos. Não que precise mudar aqui mas senti falta de algum tipo de sinalização quando for configurado um montante de pagamento/recebimento superior ao valor total da nota fiscal.